CONTRIBUTING.md

```diff
@@ -81,8 +80,8 @@ facilitates social contribution, easy testing and peer review.
 To contribute a patch, the workflow is as follows:
 
   1. Fork repository ([only for the first time](https://help.github.com/en/articles/fork-a-repo))
-  1. Create topic branch
-  1. Commit patches
+  2. Create topic branch
```

Comment:
I think the repeated `1.` is intentional to let GitHub auto-number the items. Otherwise adding a single new item would require re-numbering all subsequent items manually :)